test/integration/smoke.test.ts

```diff
@@ -88,7 +88,7 @@ describe('API Client Integration Tests', () => {
         tableName: 'table'
       },
       body: {
-        record: { email: 'example@foo.bar' }
+        email: 'example@foo.bar'
```

Comment:
We may have had a bug here and didn't notice. I don't think this part of the API was changed

Reply:
I just added this test on Friday, weird I didn't catch it